rag_experiment_accelerator/run/querying.py

```diff
@@ -150,24 +158,123 @@ def query_and_eval_acs(
         evaluation_content (str): The content to use for evaluating the search results.
         retrieve_num_of_documents (int): The number of documents to retrieve from the search results.
         evaluator (SpacyEvaluator): The evaluator to use for evaluating the search results.
+        config (Config): The configuration object.
+        response_generator (ResponseGenerator): The response generator object.
 
     Returns:
         tuple[list[dict[str, any]], dict[str, any]]: A tuple containing the retrieved documents and the evaluation results.
     """
-    search_result = query_acs(
-        search_client=search_client,
-        embedding_model=embedding_model,
-        user_prompt=query,
-        s_v=search_type,
-        retrieve_num_of_documents=retrieve_num_of_documents,
-    )
+    if config.QUERY_EXPANSION == "generated_hypothetical_answer":
```

Comment:
to avoid issues with casing I suggest you compare the lower case value of this string

Reply:
maybe do this when reading the config?

Reply:
fixed, can we resolve this @martinpeck ?